wordpress-mobile/WordPress-iOS
# Quick Start: Add progress storage

Refs #9447

This PR adds basic storage of quick start progress.

![quick start storage added](https://user-images.githubusercontent.com/517257/43227542-3aceb780-901c-11e8-83ee-8b0e55b4bce8.gif)

_Notes_:
This is just basic core data changes and UI to demonstrate them. Checkmarks UI will come in a future PR. Also, the core data code may be refactored to a new class, for now I've left it as the minimum that will work. Let me know if that sickens you 😉 

To test:
- From blog details, tap quick start
- Tap a tour
- Return to the quick start checklist
- Ensure the tour that was tapped is now struckthrough

@astralbodies can you make sure these Core Data changes aren't terrible?
